sdk/lib/ucrt/CMakeLists.txt

```diff
@@ -0,0 +1,102 @@
+
+set(CMAKE_CXX_STANDARD 17)
+
+# Replace the old CRT include directory with the UCRT include directory
+get_property(INCLUDE_DIRS DIRECTORY . PROPERTY INCLUDE_DIRECTORIES)
+list(REMOVE_ITEM INCLUDE_DIRS "${REACTOS_SOURCE_DIR}/sdk/include/crt")
+set_property(DIRECTORY . PROPERTY INCLUDE_DIRECTORIES ${INCLUDE_DIRS})
+include_directories(${REACTOS_SOURCE_DIR}/sdk/include/ucrt)
+
+if(MSVC)
+    # Disable warning C4083: expected ')'; found identifier '<warning identifier>'
+    add_compile_options($<$<COMPILE_LANGUAGE:C,CXX>:/wd4083>)
+
+    # Disable warning C4189: 'cvt': local variable is initialized but not referenced
+    add_compile_options($<$<COMPILE_LANGUAGE:C,CXX>:/wd4189>)
+endif()
+
+# Internal includes
+include_directories(BEFORE inc)
+
+if(${ARCH} STREQUAL "i386")
+    include_directories(inc/i386)
+endif()
+
+remove_definitions(-D_WIN32_WINNT=0x502 -DWINVER=0x502)
+add_compile_definitions(
+    WINVER=0x600
+    _WIN32_WINNT=0x600
+    _UCRT
+    _CORECRT_BUILD
+    _CRT_DECLARE_GLOBAL_VARIABLES_DIRECTLY
+    _GCC_NO_SAL_ATTRIIBUTES
+    CRTDLL
```

Comment:
Yeah, and that is what we are doing here. It's an object library, but the only use is to link it into ucrtbase. There is no real static version of the library, since we have no use for it.